Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -5023,3 +5024,21 @@ void CClientVehicle::ResetWheelScale()
 
     m_bWheelScaleChanged = false;
 }
+
+CVector CClientVehicle::GetEntryPoint(std::uint32_t entryPointIndex)
+{
+    static const uint32_t lookup[4] = {10, 8, 11, 9};
+    assert(entryPointIndex < 4);
+    const uint32_t saDoorIndex = lookup[entryPointIndex];
+
+    CVector      entryPoint;
+
+    if (saDoorIndex != 0)
```

Comment:
We're using the lookup table now. So `saDoorIndex` cannot not be zero in any case. This conditional can be removed with no regrets.